app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb

```diff
@@ -0,0 +1,48 @@
+<% alt_text = "" %>
+
+<svg class="covid__logos" width="300" height="45" viewBox="0 0 300 45" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="covid-logo__title">
```

Comment:
As this file only contains SVG markup should it have a file name of _stay-home-logo.svg.erb ?

Reply:
good shout 👍

Reply:
Done @kevindew